mods/cnc/mod.yaml

```diff
@@ -29,13 +18,22 @@ Packages:
 	temperat.mix
 	winter.mix
 	desert.mix
-	snow.mix
 	~movies-gdi.mix
 	~movies-nod.mix
 	~movies.mix
 	~scores.mix
 	~scores2.mix
 	~transit.mix
+	./mods/cnc/bits/snow.mix
```

Comment:
This is still mounted before `./mods/cnc/bits` and probably shouldn't have this prefix?